bitnami/etcd/templates/preupgrade-hook-job.yaml

```diff
@@ -0,0 +1,180 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+apiVersion: {{ include "common.capabilities.job.apiVersion" . }}
+kind: Job
+metadata:
+  name: {{ include "common.names.fullname" . }}-pre-upgrade
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+    app.kubernetes.io/component: etcd-pre-upgrade-job
+  annotations: {{- include "common.tplvalues.render" (dict
+    "value" (merge 
+      .Values.commonAnnotations
+      (dict
+        "helm.sh/hook" "pre-upgrade"
+        "helm.sh/hook-delete-policy" "before-hook-creation")
+    )
+    "context" $) | nindent 4 }}
+spec:
+  {{- $podLabels := include "common.tplvalues.merge" ( dict "values" ( list .Values.podLabels .Values.commonLabels ) "context" . ) }}
+  template:
+    metadata:
+      name: {{ include "common.names.fullname" . }}-pre-upgrade
```

Comment:
Redundant field (you're using the same name used for the job):
```suggestion
```

Reply:
Removed